Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: help
```

Comment:
Seems weird that we didn't have a Makefile before.

Reply:
For a while we tried to do without one for packages using nox, but it's really useful for bootstrapping and updating dependencies.